src/extensions/filter-control/utils.js

```diff
@@ -73,56 +93,54 @@ export function sortSelectControl (selectControl, orderBy) {
   $selectControl.append($opts)
 }
 
-export function fixHeaderCSS ({ $tableHeader }) {
-  $tableHeader.css('height', '89px')
+export function fixHeaderCSS ({ $tableHeader }, pixels = '89px') {
```

Comment:
I'm not sure if the bootstrap-table still works with the internet explorer, but in the [browserslistrc](https://github.com/wenzhixin/bootstrap-table/blob/develop/.browserslistrc) we have `Explorer >= 10`.
Default parameter values are not supported in IE, but in my opinion we can drop the support of IE as only 1% of the users are IE users.

Reply:
I also think that we should drop the IE's support.